src/showcase/stories/angular/sidebar/sidebar-close-button/sidebar-close-button.stories.ts

```diff
@@ -1,9 +1,48 @@
 import { SbbSidebarCloseButton } from '@sbb-esta/lyne-angular/sidebar/sidebar-close-button';
 import { Args, argsToTemplate, Meta } from '@storybook/angular';
+import { ArgTypes, InputType, StoryContext } from '@storybook/types';
+
+const size: InputType = {
```

Comment:
I'd prefer not have any of this controls, as they don't make sense at all for the close button. Is it really needed?

Reply:
see other comment https://github.com/sbb-design-systems/lyne-angular/pull/48#discussion_r2016890759